app/views/record/_availability.html.erb

```diff
@@ -1,15 +1,26 @@
 <h2 class="subtitle2">Availability</h2>
 <% if @record.fulltext_link.present? %>
   <div class="link-fulltext">
+
+    <%# SFX links that we don't know for sure if we own; auth happens at SFX %>
```

Comment:
This is starting to feel like a lot of logic for a view. I don't have a better idea, mind you, but I feel twitchy.